builtin/logical/transit/path_derive_key.go

```diff
@@ -0,0 +1,165 @@
+// Copyright (c) HashiCorp, Inc.
```

Comment:
```suggestion
// Copyright (c) 2025 OpenBao a Series of LF Projects, LLC
```

This is net-new code that you've introduced.